Jenkinsfile

```diff
@@ -569,6 +569,31 @@ pipeline {
                  }
              }
         }
+        stage('Self-driving end-to-end test') {
+            agent { label 'benchmark' }
```

Comment:
How much speed different are there between the benchmark machines and other machines? In general it's better to assign a reasonable machine to the self-driving test because the runners would consume some time to generate the training data.

Reply:
The benchmark machines are actually not necessarily the strongest machines, e.g., 

- benchmark: 2.40GHz, 24 CPUs
- newer swarm machine: 3.10GHz, 4 CPUs

What makes the benchmark machines special is that jobs get exclusive use of the machine.
This helps (but also not really) to reduce variance in End-To-End Performance.